src/pe_reports/stakeholder/views.py

```diff
@@ -0,0 +1,595 @@
+
+            cursor.execute(f"insert into organizations(name)" f"values('{customer}')")
+
+            return True
+
+    except (Exception, psycopg2.DatabaseError) as err:
+        logging.error("There was a problem logging into the psycopg database %s", err)
+        return False
+    finally:
+        if conn is not None:
+            conn.commit()
+            cursor.close()
+            conn.close()
+            logging.info("The connection/query was completed and closed.")
+
+
+def setCustRootDomain(customer, rootdomain, orgUUID):
+    """Insert customer into the PE-Reports database."""
+    global conn, cursor
+
+    try:
+        logging.info("Starting insert into database...")
+
+        params = config()
+
+        conn = psycopg2.connect(**params)
+
+        if conn:
+
+            logging.info(
+                "There was a connection made to "
+                "the database and the query was executed "
+            )
+
+            cursor = conn.cursor()
+
+            cursor.execute(
+                f"insert into root_domains("
+                f"organizations_uid,"
+                f"organization_name,"
+                f" root_domain)"
+                f"values('{orgUUID}', '{customer}','{rootdomain}');"
+            )
+            return True
+
+    except (Exception, psycopg2.DatabaseError) as err:
+        logging.error("There was a problem logging into the psycopg database %s", err)
+        return False
+    finally:
+        if conn is not None:
+            conn.commit()
+            cursor.close()
+            conn.close()
+            logging.info("The connection/query was completed and closed.")
+
+
+def setCustSubDomain(subdomain, rootUUID, rootname):
+    """Insert customer into the PE-Reports database."""
+    global conn, cursor
+
+    try:
+
+        logging.info("Starting insert into database...")
+
+        params = config()
+
+        conn = psycopg2.connect(**params)
+
+        if conn:
+
+            logging.info(
+                "There was a connection made to "
+                "the database and the query to "
+                "insert the subdomains was executed "
+            )
+
+            cursor = conn.cursor()
+
+            for sub in subdomain:
+                cursor.execute(
+                    f"insert into sub_domains("
+                    f"sub_domain,"
+                    f"root_domain_uid,"
+                    f" root_domain)"
+                    f"values('{sub}',"
+                    f" '{rootUUID}',"
+                    f"'{rootname}');"
+                )
+            return True
+
+    except (Exception, psycopg2.DatabaseError) as err:
+        logging.error("There was a problem logging into the psycopg database %s", err)
+        return False
+    finally:
+        if conn is not None:
+            conn.commit()
+            cursor.close()
+            conn.close()
+            logging.info("The connection/query was completed and closed.")
+
+
+def setCustomerExteralCSG(
```

Comment:
Typo?  If so, be sure to correct this name wherever else it is used.
```suggestion
def setCustomerExternalCSG(
```

Reply:
@dav3r , 
I accepted this change and will make the updates where the function is used.